controller/linodemachine_controller.go

```diff
@@ -62,6 +62,7 @@ const (
 	// conditions for preflight instance creation
 	ConditionPreflightBootstrapDataSecretReady  clusterv1.ConditionType = "PreflightBootstrapDataSecretReady"
 	ConditionPreflightLinodeFirewallReady       clusterv1.ConditionType = "PreflightLinodeFirewallReady"
+	ConditionPreflightLinodeNBFirewallReady     clusterv1.ConditionType = "PreflightLinodeNBFirewallReady"
```

Comment:
Can we move this to here? https://github.com/linode/cluster-api-provider-linode/blob/main/controller/linodecluster_controller.go#L56

Reply:
Done!